src/exception.rs

```diff
@@ -103,6 +103,11 @@ pub fn handle_exception_sync(ctx: &mut TrapFrame) -> AxResult<AxVCpuExitReason>
             // The `#imm`` argument when triggering a hvc call, currently not used.
             let _hvc_arg_imm16 = ESR_EL2.read(ESR_EL2::ISS);
 
+            // Is this a psci call?
```

Comment:
More things need to there.
Guest maybe do smc call to handle PSCI.
In linux, you can give it a dtb.
```
        psci {
		migrate = <0xc4000005>;
		cpu_on = <0xc4000003>;
		cpu_off = <0x84000002>;
		cpu_suspend = <0xc4000001>;
		method = "smc";
		compatible = "arm,psci-1.0\0arm,psci-0.2\0arm,psci";
	};
```

Reply:
Yep, but PSCI related device info should be managed elsewhere.